app/poros/quote.rb

```diff
@@ -0,0 +1,10 @@
+class Quote
+  attr_reader :id, :content, :author, :tags
+
+  def initialize(attributes)
+    @id = attributes[:_id]
```

Comment:
When I ran your tests, line 5 failed with this error:
```
1) Quote PORO creates a quote object
     Failure/Error: @id = attributes[:_id]

     TypeError:
       no implicit conversion of Symbol into Integer
     # ./app/poros/quote.rb:5:in `initialize'
     # ./spec/poros/quote_spec.rb:8:in `new'
```
I don't even think that you need `:id`. It isn't one of the attributes that we need.